package.json

```diff
@@ -41,13 +41,13 @@
     "postcss-scss": "^2.0.0",
     "prettier": "^1.19.1",
     "rimraf": "^3.0.0",
-    "stylelint": "^12.0.0"
+    "stylelint": "^13.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
+    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
```

Comment:
BTW this is getting too long. We should probably drop some versions from here.

Reply:
Can we use the [range](https://docs.npmjs.com/misc/semver#ranges) operator `>=`?

```
"stylelint": ">=8.0.0"
```